debian.master/config/config.common.ubuntu

```diff
@@ -9842,7 +9842,7 @@ CONFIG_SND_AUDIO_GRAPH_CARD=m
 CONFIG_SND_AW2=m
 CONFIG_SND_AZT3328=m
 CONFIG_SND_BCD2000=m
-# CONFIG_SND_BCM2835 is not set
+CONFIG_SND_BCM2835=m
```

Comment:
Why are you also re-enabling BCM2835? (Please also explain it on the commit message)

Reply:
Because CONFIG_SND_BCM2835 is a sub item of CONFIG_BCM_VIDEOCORE,
disabling CONFIG_BCM_VIDEOCORE will disable CONFIG_SND_BCM2835 as well.
So, partially drop the change of CONFIG_SND_BCM2835 to have less
downstream modification.

Updated to the commit message as well.